packages/angular/cli/commands/doc.json

```diff
@@ -29,14 +29,11 @@
         "version" : {
           "anyOf": [
             {
-              "const" : 2
-            },
-            {
-              "type" : "integer",
+              "type": "number",
               "minimum": 4
             },
             {
-              "const": "next"
+              "enum": [2, "next"]
```

Comment:
I am not understanding this `2` here, what is the reason behind `2` being accepted?

Reply:
https://v2.angular.io